sst.config.ts

```diff
@@ -0,0 +1,131 @@
+/// <reference path="./.sst/platform/config.d.ts" />
+import { remote, types } from "@pulumi/command";
+
+export default $config({
+	app(input) {
+		return {
+			name: "fourity-plants",
+			removal: input?.stage === "production" ? "retain" : "remove",
+			protect: ["production"].includes(input?.stage),
+			home: "aws",
+			region: "us-east-1",
+		};
+	},
+	async run() {
+		const aws = await import("@pulumi/aws");
+		const awsx = await import("@pulumi/awsx");
+		const docker = await import("@pulumi/docker");
+		const { remote, types } = await import("@pulumi/command");
+		const pulumi = await import("@pulumi/pulumi");
+
+		// const image = new awsx.ecr.Image("image", {
+		// 	repositoryUrl:
+		// 		"351931932329.dkr.ecr.eu-central-1.amazonaws.com/fourity/fourity-plants",
+		// 	context: "./docker",
+		// 	platform: "linux/amd64",
+		// });
+
+		// const config = new pulumi.Config();
+
+		// const userName = config.require("fourity"); //userName
+		// const serverPublicIp = config.require("192.168.0.58"); //ip of the server
+		// const privatekey = config.requireSecret("privateKey"); //SSH i have in folder
+
+		// const connection: types.input.remote.ConnectionArgs = {
+		// 	host: serverPublicIp,
+		// 	user: userName,
+		// 	privateKey: privatekey,
+		// };
+
+		// const push = new remote.Command(
+		// 	"docker",
+		// 	{
+		// 		connection: connection,
+		// 		create: "pulling of docker image",
+		// 		delete: "removal of docker image",
+		// 		triggers: [image.imageUri],
+		// 	},
+		// 	{ dependsOn: image },
+		// );
+
+		const zone_table = new aws.dynamodb.Table("zone-table", {
```

Comment:
We use SST as the main provisioner and only fallback to pulumi if we need it.
in this case ```sst.aws.dynamo``` exists.

In order to attach additional params which are not available in SST TS auto-complete, we use the following:
```ts
transform: {
  function(args) {
    args.billingMode = "PROVISIONED"
    ...
  }
}
```